pkg/cassandra/config/config.go

```diff
@@ -38,6 +38,7 @@ type Configuration struct {
 	MaxRetryAttempts     int           `validate:"min=0" yaml:"max_retry_attempt"`
 	ProtoVersion         int           `yaml:"proto_version"`
 	Consistency          string        `yaml:"consistency"`
+	EnableCompression    bool          `yaml:"enable-compression"`
```

Comment:
This makes the struct not backwards compatible, because the default value will be false (only initializing from CLI flags would use true as default).

I prefer to call this `DisableCompression`

Reply:
@yurishkuro : Sure, disableCompression makes more sense, now that I think about it. Will make the changes. 
Quick question though, the following puts the default value of enableCompression to be true. So, default as true also affects the backend compatibility? 
```suggestion
	flagSet.Bool(
		nsConfig.namespace+suffixEnableCompression,
		true,
		"Enable the use of Compression while connecting to Cassandra API based Storage backend. Uses SnappyCompression by default")
```

Reply:
This default only works if flags are used. But the CassandraConfiguration struct can also be used directly, e.g. it's how we do it in our customized binaries at Uber, where configuration is read from yaml files and auto-populated into config structs. So the struct itself should be backwards compatible.

Reply:
Thank you for this @yurishkuro 
Made the required changes, please review.